server/models/Product.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose")
+
+const ProductSchema = new mongoose.Schema(
```

Comment:
As I understand it, the schema should be:
id
title
category
description
condition
price
imageURLs (list)
postDate

I wonder if we might also want to add a customerID/username field for the customer who posted it

Reply:
We can add customerID if we want later on. I think that it can also be related to our login ID

Reply:
True! Sounds good